nibabel/streamlines/compact_list.py

```diff
@@ -0,0 +1,232 @@
+import numpy as np
+
+
+class CompactList(object):
+    """ Class for compacting list of ndarrays with matching shape except for
```

Comment:
Sorry - should have said - PEP8 sets the maximum line length to 79 characters : https://www.python.org/dev/peps/pep-0008/#maximum-line-length . So, your job, should you choose to accept it, is to find a nice compact way of summarizing the function (or class or method) withing 79 characters !